app/build.gradle

```diff
@@ -97,29 +97,35 @@ android {
 }
 
 dependencies {
+    def composeBom = platform(libs.compose.bom)
```

Comment:
this would've been better as a separate PR as it makes it hard/impossible to compare

Reply:
I can group the bomUsages in one block, so it's easier to review – WDYT?